Wikipedia/Code/ShareViewController.swift

```diff
@@ -55,6 +55,16 @@ class ShareViewController: UIViewController, Themeable {
             self.showActivityViewController(with: image)
         }
     }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        coordinator.animate(alongsideTransition: { (context) in
+            if let presentedActivityVC = self.presentedViewController as? UIActivityViewController {
```

Comment:
One thing I noticed re: rotation on the iPhone 11 Pro Simulator:

1. Start in portrait mode and waiting for share card to animate to top 
2. Rotate to landscape 
3. Rotate back to portrait

The share card returns to the center of the view instead of staying pinned to the top and is covered by the system share sheet. It might be `imageViewVerticallyCenteredConstraint` related, so one possible solution may be something like adding this here in `coordinator.animate(alongsideTransition: ...)`:
```swift
self.imageViewTopConstraint.isActive = true
self.imageViewVerticallyCenteredConstraint.isActive = false
```

![iphone11-share](https://user-images.githubusercontent.com/5652327/84328608-448f1480-ab37-11ea-9e87-6ad1814cf590.png)

Reply:
That fix does it. I spent a while digging in trying to figure out another way. It seems like installed/uninstalled constraints are guaranteed to remain [constant upon rotation](https://stackoverflow.com/questions/13739902/ios-change-auto-layout-constraints-when-device-rotates), but I can't find anything from Apple saying this. But it's definitely what's happening here. Further, I think it's just being reset when class sizes change - as on rotation on iPad (regular to regular) the problem doesn't occur. (It does occur when moving from split screen to full screen on iPad, which is more evidence that it's specifically related to class sizes. Working statement: When class sizes change on iOS, installed/uninstalled constraints revert to what exists on the xib.)

TL;DR: Now fixed, ready for another review.

Reply:
Awesome – glad it was a small fix for such a strange issue.